File: python/openEMS/physical_constants.py
# -*- coding: utf-8 -*-
"""
Created on Fri Dec 18 20:58:01 2015

@author: thorsten
"""

import numpy as np

C0 = 299792458 # m/s
MUE0 = 4e-7*np.pi # N/A^2
EPS0 = 1/(MUE0*C0**2) # F/m

# free space wave impedance
Z0 = np.sqrt(MUE0/EPS0) # Ohm

File: matlab/physical_constants.m
%
% physical constants
%

% Bronstein 3rd ed., 1997, pp. 945-946
C0 = 299792458; % m/s
c0 = C0; % constants in capital letters, c0 for legacy support
MUE0 = 4e-7*pi; % N/A^2
EPS0 = 1/(MUE0*C0^2); % F/m

% free space wave impedance
Z0 = sqrt(MUE0/EPS0); % Ohm
